managed/services/supervisord/logs_test.go

```diff
@@ -151,7 +169,7 @@ func TestFiles(t *testing.T) {
 	l := NewLogs("2.4.5", checker, params)
 	ctx := logger.Set(context.Background(), t.Name())
 
-	files := l.files(ctx, nil)
+	files := l.files(ctx, nil, -1)
```

Comment:
Well, the default is 50K lines and there is no way to know when the log files will be large enough so we can check if the actual line count is equal to 50K lines. I have tested it manually and the result was OK. 

Unless, of course, you meant passing `0`, which means `use the default maxLineCount`. But even then the outcome would be the same. 

Please let me know your preference )

Reply:
Update: we are passing `0` now.

Reply:
no I meant that since default behavior for this feature is default value, not unlimited it's better to use default value if we don't test anything else.